### ISSUE TITLE
Sorting and/or Tagging Studies in Admin Review Queue - MKS to parse out

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#863]
**TL;DR**: When looking at the list of studies submitted for Admin Review (and approval/rejection), I want better ways to keep track of information regarding original date of submission, whether I have scheduled a video chat session (for synchronous external studies), etc.

**Narrative**
As an admin or study reviewer, I am losing track of which studies in the admin review queue are the highest priority for my attention. This is because we have many more studies, and the displayed information does not capture the information I need to make my decision. Two things stand out as most important: (1) I can see "Last Edited" but NOT when it was originally submitted for admin review--this means that I may think a study is low priority rather than very high priority (e.g., I think it was only submitted yesterday, when in fact it was submitted two weeks ago but the date got updated when a researcher made a trivial change to it), (2) now that we have external synchronous studies, I might schedule a live review session (e.g., on Zoom) at the researcher's earliest available time 2 weeks in the future, and now there is nothing more for me to do with it in the queue, but it will capture my attention as being "old" and demanding my attention.

**Implementation Notes**
- I think the first point (1) could be addressed by having the "Study submitted" visible in the summary view I see when looking at Manage Studies: Submitted AND having the list sortable by that field
- I think (1) is trickier and has more potential solutions, and I'm not sure how difficult different ones would be to implement. One idea that comes to mind is having a field I can toggle to indicate a live admin review session is scheduled, and then it moves to a different tab OR to the bottom of the list.

**Comment by mekline** [comment]
Closed to reflect work done in #991, and remaining/not done in #1019
